Tools/translate.cmd

```diff
@@ -0,0 +1,9 @@
+rem Create the R-nl language .mo file
+
+rem %1 is Rscript location
+rem %2 is get gettext location (stripped "")
+rem %3 Tools location
+rem %4 R source folder
+
+set PATH=%PATH%;%~2
+%1\Rscript %3\translate.R %4
```

Comment:
So why is it a bat file on windows but a call on unix?
This is exactly the same commands right?

Or well, I guess the temporary env trick I alluded to earlier doesn't work on Windows...

Reply:
Yes the reason is  the env annoying thing windows/Qt

Reply:
Or you want to have the R file because then you can use it from here? Because that isn't a bad reason actually, Windows is annoying indeed.

Reply:
You think I can merge this one after last changes? Then I can try to find the SIMPLE_CRYPT issue  :-)